onboarding-new-staff.md

```diff
@@ -104,24 +104,21 @@ Once the team member has had their hardware provisioned, the next stage is to go
  - Talk through the key team members
    - Directors
      - Andy Greener (MD)
-     - Armin Talic (CD)
      - Jason Christie (PD)
-     - Ian Outterside (TD)
    - Management
      - Chris Jones (Client Services)
      - Phoebe Dowley (Account Management)
      - Nicola O'Neill (Office Management)
    - Teams & Leads
      - Design
-       - James Tabiner - aka Tabs (Senior Designer)
        - Thomas Wood - aka Woody (Senior Designer)
      - Development
        - PHP - Laravel, Symfony, WordPress
-         - Scott Salisbury (Senior Developer)
+         - Scott Salisbury (Head of Web Development)
        - Frontend / JS - React / Node
-         - Chris Neale (Senior Developer)
+         - Daniel Carney (Senior Developer)
```

Comment:
frontend 💀

Reply:
wouldn't say i'm a team leader for frontend, better off just removing that section i reckon or putting flowers